src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const extraDiscount = 50;
+  const rentCosts = 40;
```

Comment:
You can place this variables outside of the function.
Now they are created anew with each new function call.